Formula/r/reflex.rb

```diff
@@ -0,0 +1,18 @@
+class Reflex < Formula
+  desc "Run a command when files change"
+  homepage "https://github.com/cespare/reflex"
+  url "https://github.com/cespare/reflex.git",
+    tag:      "v0.3.1",
+    revision: "13e5691dcde5f7c29c144d1cb8c34f453d78505d"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    assert_match "Could not make reflex for config: must give command to execute", shell_output("#{bin}/reflex", 1)
```

Comment:
From the log, looks like the error text is still ending up in the log rather than being captured. This usually happens since `shell_output` captures only `stdout` and the text may be getting sent to `stderr`. The usual shell redirect `2>&1` should work here.

```suggestion
    assert_match "Could not make reflex for config: must give command to execute", shell_output("#{bin}/reflex 2>&1", 1)
```